thunder/tests/test_examine_memory.py

```diff
@@ -156,16 +149,16 @@ def test_nanogpt_block(executor, device, dtype):
 
     if isinstance(executor, nvFuserTestExecutor):
         assert fw_alloc_mem[0] == 267426816
-        # t67 is the expand result of ln_2_weight, and they are both return values in trace
-        # but for calculation we assume they share memory, so expect to subtract the size of t67
-        expected_return_calculated_mem = get_return_memory(fw_extrace.bound_symbols[-1]) - 4 * 2 * 1024 * 768
+        expected_return_calculated_mem = get_return_memory(fw_extrace.bound_symbols[-1])
         assert expected_return_calculated_mem == sum(fw_alloc_mem[1].values())
 
-        assert bw_alloc_mem[0] == 361783296
+        assert bw_alloc_mem[0] == 412112896
```

Comment:
The increase is again due to the limitation of `get_alloc_memory`. It assumes all outputs of an nvFusion are allocated. However, outputs are sometimes aliases.